src/SecurityHubOverviewLambda/SecurityHubOverview-function.ts

```diff
@@ -0,0 +1,26 @@
+// ~~ Generated by projen. To modify, edit .projenrc.js and run "npx projen".
+import * as path from 'path';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import { Construct } from 'constructs';
+
+/**
+ * Props for SecurityHubOverviewFunction
```

Comment:
Dit soort comments mogen van mij weg. Voegt niks toe t.o.v. de naam van de interface.

Reply:
Deze class is gegenereerd door projen :)